modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceConfigurationManager.java

```diff
@@ -133,11 +133,11 @@ public ServiceOrder changeServiceConfiguration(String serviceId,
         try {
             ServiceDeploymentEntity serviceDeploymentEntity = serviceDeploymentEntityHandler
                     .getServiceDeploymentEntity(UUID.fromString(serviceId));
-            ServiceTemplateEntity serviceTemplateEntity = serviceTemplateStorage
-                    .getServiceTemplateById(serviceDeploymentEntity.getServiceTemplateId());
+            ServiceTemplateEntity serviceTemplateEntity =
```

Comment:
for all methods in this class, it is better to change the type of serviceId to UUID instead of string.